lib/perl/Genome/Sys/Lock.pm

```diff
@@ -151,30 +159,63 @@ sub with_default_lock_resource_args {
 
     $args{block_sleep} = 60 unless defined $args{block_sleep};
     $args{max_try} = 7200 unless defined $args{max_try};
+    $args{scope} = $ENV{GENOME_SYS_LOCK_DEFAULT_SCOPE} unless defined $args{scope};
```

Comment:
Yeah, I think this is possibly true.  I propose below that we eliminate the environment variable altogether.

Maybe we should change the check for `GENOME_*` environment variables into a warning instead of an error?